### ISSUE TITLE
Feature Request | Date/Time Packer bottom spinner (native iPhone style date/time picker)

### ISSUE THREAD

**Issue by moriswala** [GiancarloCode/form_bloc#108]
Please if you can provide Date/Time picker look like native iPhone (bottom spinner) would be great.

Additionally, current Date picker does not provide hijri dates. It would be great if you can keep option for hijri date in existing as well as future date picker widget.

Thank you

**Comment by ignaciolopez12** [comment]
I am looking for the same, have you found any solution?

**Comment by moriswala** [comment]
@ignaciolopez12 I decided to stop using this as it has some limitations and used [Flutter](https://pub.dev/packages/flutter_bloc) as It is more flexible. and for Hijri date picker I used https://pub.dev/packages/hijri_picker.

**Comment by GiancarloCode** [comment]
@moriswala  @ignaciolopez12 
Hello, sorry for the delay, you can use the widget you want easily,
check [this comment](https://github.com/GiancarloCode/form_bloc/issues/64#issuecomment-613836269) that explains how to do it

**Comment by ignaciolopez12** [comment]
> @moriswala @ignaciolopez12
> Hello, sorry for the delay, you can use the widget you want easily,
> check [this comment](https://github.com/GiancarloCode/form_bloc/issues/64#issuecomment-613836269) that explains how to do it


Thank you! I found out that flutter_bloc was updated to version 6 ... flutter_form_bloc for it to work well, should we expect it to launch a new version too?

**Comment by GiancarloCode** [comment]
@ignaciolopez12 Yes, I will check some pull requests before so that they are not outdated and update tomorrow or the day after tomorrow

**Comment by ignaciolopez12** [comment]
> @ignaciolopez12 Yes, I will check some pull requests before so that they are not outdated and update tomorrow or the day after tomorrow


Excellent! Thank you very much for your contributions!
